docs/source/API/core/atomics/atomic_load.rst

```diff
@@ -4,22 +4,27 @@
 .. role:: cppkokkos(code)
     :language: cppkokkos
 
-Header File: ``<Kokkos_Core.hpp>``
+Defined in header ``<Kokkos_Atomic.hpp>`` which is included from ``<Kokkos_Core.hpp>``
```

Comment:
Just curious about the wording, does this mean that the user should use `#include <Kokkos_Core.hpp>` instead of `#include <Kokkos_Atomic.hpp>`, or is it just a way to tell people not to bother if they already include `Kokkos_Core.hpp`?

Reply:
The latter